build.gradle

```diff
@@ -72,42 +73,67 @@ dependencies {
 	testCompile 'org.mockito:mockito-all:2.0.2-beta'
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: ':shadowJar') {
+def assetsDirectory = file("${buildDir}/assets")
+git {
+	assets {
+		url 'https://github.com/triplea-game/assets.git'
+		directory assetsDirectory
+		branch 'master'
+		singleBranch true
+	}
+}
+import com.install4j.gradle.Install4jTask
+
+task updateAssets(dependsOn: ['gitCloneAssets', 'gitPullAssets']) {
```

Comment:
Why the empty task? If something depends on 'updateAssets', shouldn't we just have it depend on 'gitClone' and 'gitPullAssets'? Is it best practice to bundle these dependencies like this or something?